Gordon360/Controllers/RecIMControllers/LeagueController.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+
+namespace Gordon360.Controllers.RecIMControllers
+{
+    [Route("api/recim")]
+    public class LeagueController : GordonControllerBase
+    {
+        private readonly ILeagueService _leagueService;
+
+        public LeagueController(ILeagueService leagueService)
+        {
+            _leagueService = leagueService;
+        }
+
+        ///<summary>Gets a list of all Leagues</summary>
+        /// <returns>
+        /// All Existing Leagues 
+        /// </returns>
+        [HttpGet]
+        [Route("league")]
```

Comment:
Since you can add the controller name to the controller-wide route prefix, you don't need to specify it here. Also, the route should be "/leagues", as specified above.
```suggestion
        [Route("")]
```

Reply:
I updated all the routes to ignore league and added a forward slash